elements/lisk-p2p/test/utils/network_setup.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { P2P } from '../../src/index';
+import { wait } from './helpers';
+import { platform } from 'os';
+import {
+	DEFAULT_MAX_OUTBOUND_CONNECTIONS,
+	DEFAULT_MAX_INBOUND_CONNECTIONS,
+} from '../../src';
+export const NETWORK_START_PORT = 5000;
+export const NETWORK_PEER_COUNT = 10;
+export const POPULATOR_INTERVAL = 50;
+export const DEFAULT_CONNECTION_TIMEOUT = 500;
+export const DEFAULT_ACK_TIMEOUT = 500;
+export const RATE_CALCULATION_INTERVAL = 10000;
+export const WEB_SOCKET_ENGINE = 'ws';
+export const SEED_PEER_IP = '127.0.0.1';
+export const NETWORK_CREATION_WAIT_TIME = 1000;
+export const NETWORK_DESTROY_WAIT_TIME = 1000;
+
+export const nodeInfoConstants = {
+	nethash: 'da3ed6a45429278bac2666961289ca17ad86595d33b31037615d4b8e8f158bba',
+	version: '1.0.1',
+	protocolVersion: '1.1',
+	minVersion: '1.0.0',
+	os: platform(),
+	height: 0,
+	broadhash: '2768b267ae621a9ed3b3034e2e8a1bed40895c621bbb1bbd613d92b9d24e54b5',
+	nonce: `O2wTkjqplHII`,
+};
+
+interface TestNetworkConfig {
+	networkSize?: number;
+	startNodePort?: number;
```

Comment:
Do we actually need this? Are there cases where it isn't `5000`?

Reply:
Maybe we don't need it but I see a lot of values been calculated using this value. So, in general, I wanted to make it configurable if someone wants to run two parallel integration tests on different ports then it will be helpful to modify this value.